framework/include/dampers/ConstantDamper.h

```diff
@@ -39,7 +39,8 @@ class ConstantDamper : public GeneralDamper
   /**
    * Return the constant damping value.
    */
-  virtual Real computeDamping(const NumericVector<Number> & update);
+  virtual Real computeDamping(const NumericVector<Number> & solution,
+                              const NumericVector<Number> & update);
```

Comment:
I think I've always just commented them out in the .C files, which is what I've done here.  I'm happy to comment them out in the .h files, but I don't think it's necessary to avoid compile warnings because the parameter names in the .h files aren't actually used for anything.